internal/authtoken/options.go

```diff
@@ -14,12 +25,18 @@ type Option func(*options)
 
 // options = how options are represented
 type options struct {
-	withTokenValue bool
-	withLimit      int
+	withTokenValue               bool
+	withTokenTimeToLiveDuration  time.Duration
+	withTokenTimeToStaleDuration time.Duration
+	withLimit                    int
 }
 
 func getDefaultOptions() options {
-	return options{}
+	return options{
+		withLimit:                    db.DefaultLimit,
```

Comment:
I don't think we need or want to do this. If we don't have this option set, the DB will already set this to the default. There's no reason to also have this codified here.

Reply:
Oh I see, this was already in NewRepository. I think probably that code in and of itself was already not correct. @jimlambrt thoughts? Is there a reason to set to db.DefaultLimit explicitly here or in the repo code, or will that automatically be used by the underlying db code?

Reply:
Previously, when you called NewRepo(), that would set this default if the value was zero. We agreed to move that logic into options.go getDefaultOptions(). If we don't set this here, we will have a 0 value.

Reply:
Right -- see my second comment above as to whether we actually wanted that behavior in the first place. Could use Jim's opinion here.